etc/snap.properties.bak

```diff
@@ -0,0 +1,61 @@
+####################################################################################
```

Comment:
Probably accidentally checked in, right?

Reply:
jupp